src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/WelcomeController.java

```diff
@@ -43,7 +38,7 @@
         return ok("Welcome to CCD Case Document AM Controller");
     }
 
-    @RequestMapping("/swagger")
+    @GetMapping(value = "/swagger")
     public String index() {
```

Comment:
We really need this end point.As, Swagger itself provides default end points.

Reply:
Above comments can be incorporated as part of clean up task.